src/azul/indexer.py

```diff
@@ -33,12 +33,9 @@ def index(self, dss_notification: Mapping[str, Any]) -> None:
                                                                version=bundle_version,
                                                                num_workers=config.num_dss_workers)
 
+        self.define_indexes()
```

Comment:
Could you pass `es_client` into this function?  It seems unnecessary to do `ESClientFactory.get()` in `define_indexes`